llvm/lib/Target/AMDGPU/AMDGPULegalizerInfo.cpp

```diff
@@ -4737,7 +4737,7 @@ bool AMDGPULegalizerInfo::legalizeFDIV32(MachineInstr &MI,
 
   // FIXME: This mishandles dynamic denormal mode. We need to query the
```

Comment:
This is still broken, we're just changing the mode incorrectly in the other direction